zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -2529,15 +2529,15 @@ paths:
                 $ref: "#/components/schemas/ProblemDetail"
         "500":
           $ref: "#/components/responses/InternalServerError"
-  /mapping-rules/{mappingKey}:
+  /mapping-rules/{id}:
```

```suggestion
  /mapping-rules/{mappingRuleId}:
```

🔧 Prefer qualified identifier names for clarity.